lib/datasource/index.ts

```diff
@@ -27,7 +29,10 @@ export const getDatasourceList = (): string[] => Array.from(datasources.keys());
 
 const cacheNamespace = 'datasource-releases';
 
-function load(datasource: string): DatasourceApi {
+datasources.set('cdnjs', new CdnJs());
+datasources.set('clojure', new Clojure());
```

Comment:
```suggestion
```

already done in `./api.ts`. wrong merge?

Reply:
Yes, I think this was done before before the dynamic to static migration.

Reply:
Yes, I think this was done before the dynamic to static migration